cli.js

```diff
@@ -364,6 +364,7 @@ else if (command.match(/^global$/i) && argv[1] || argv[0] && !argv[1]) {
         exit();
```

Comment:
If you unfold the code above this line, you'll find the call to n.setGlobal which is called regardless of whether the version could be resolved. Just move it somewhere after the if-guard you inserted below ;)